dts/riscv/ite/it51xxx.dtsi

```diff
@@ -1025,6 +1027,75 @@
 			interrupt-parent = <&intc>;
 		};
 
+		prs: pwmprs@f04680 {
```

Comment:
Is this node necessary? (This reg base can be listed to each pwm node below)

Reply:
Removed